dev.vencord.Vesktop.yml

```diff
@@ -0,0 +1,68 @@
+app-id: dev.vencord.Vesktop
+
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+
+command: startvesktop
+separate-locales: false
+
+finish-args:
+  - --socket=pulseaudio
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications # Discord push notifs
+  - --talk-name=org.kde.StatusNotifierWatcher # Tray functionalities on KDE
+  - --device=all # Needed for GPU acceleration and the webcam
+  - --filesystem=xdg-videos:ro
+  - --filesystem=xdg-pictures:ro
+  - --filesystem=xdg-download # This and the above two are used for drag-n-drop, and download managing
+  - --filesystem=xdg-run/pipewire-0 # Pipewire interfacing
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons # This is used to show the correct cursor on Wayland
+
+modules:
+  - name: minerals
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 minerals/startvesktop /app/bin/startvesktop
+      - install -D minerals/dev.vencord.Vesktop.desktop -t /app/share/applications/
+      - install -D dev.vencord.Vesktop.metainfo.xml -t /app/share/metainfo/
+    sources:
+      - type: dir
+        path: .
+
+  - name: Vesktop
+    buildsystem: simple
+    build-commands:
+      - chmod +x Vesktop*.AppImage
+      - ./Vesktop*.AppImage --appimage-extract
+      - install -D squashfs-root/usr/share/icons/hicolor/1024x1024/apps/vencorddesktop.png /app/share/icons/hicolor/1024x1024/apps/dev.vencord.Vesktop.png
+      - install -D squashfs-root/usr/share/icons/hicolor/512x512/apps/vencorddesktop.png /app/share/icons/hicolor/512x512/apps/dev.vencord.Vesktop.png
+      - install -D squashfs-root/usr/share/icons/hicolor/256x256/apps/vencorddesktop.png /app/share/icons/hicolor/256x256/apps/dev.vencord.Vesktop.png
+      - install -D squashfs-root/usr/share/icons/hicolor/128x128/apps/vencorddesktop.png /app/share/icons/hicolor/128x128/apps/dev.vencord.Vesktop.png
+      - install -D squashfs-root/usr/share/icons/hicolor/64x64/apps/vencorddesktop.png /app/share/icons/hicolor/64x64/apps/dev.vencord.Vesktop.png
+      - install -D squashfs-root/usr/share/icons/hicolor/48x48/apps/vencorddesktop.png /app/share/icons/hicolor/48x48/apps/dev.vencord.Vesktop.png
+      - install -D squashfs-root/usr/share/icons/hicolor/32x32/apps/vencorddesktop.png /app/share/icons/hicolor/32x32/apps/dev.vencord.Vesktop.png
+      - install -D squashfs-root/usr/share/icons/hicolor/16x16/apps/vencorddesktop.png /app/share/icons/hicolor/16x16/apps/dev.vencord.Vesktop.png
```

Comment:
```suggestion
```

Just install 512px and 16px if you need it for tray. Flathub only needs 128px png or svg. It'll generate one from the 512px